src/store/WalletStore.ts

```diff
@@ -34,6 +35,22 @@ export default defineStore("WalletStore", {
 		setNetworkTypes(network: NetworkTypes) {
 			console.log("Network Changed: ", network);
 			this.network = network;
+			this.setAlgoExplorerUrl();
+		},
+		setAlgoExplorerUrl() {
+			switch (this.network) {
+				case NetworkTypes.MAIN_NET:
+					this.algoExplorerURL =
+						"https://algoindexer.algoexplorerapi.io/v2/transactions/";
+					break;
+				case NetworkTypes.TEST_NET:
+					this.algoExplorerURL =
+						"https://algoindexer.testnet.algoexplorerapi.io/v2/transactions/";
+					break;
+			}
+		},
```

Comment:
I thought we already have it in our wallet widget.

Reply:
we have the url in web library for node not for indexer